README.rdoc

```diff
@@ -40,6 +40,11 @@ objects has these attributes:
 * *rating*: the rating of this spot on Google Places
 * *url*: the url of this spot on Google Places
 
+However <tt>address_components</tt>, <tt>city</tt>, <tt>country</tt>, <tt>formatted_address</tt>, <tt>region</tt> and <tt>url</tt> are <tt>nil</tt>.
+
+* To get these values: You can use <tt>@client.spot(place_id)</tt>. This returns more information of the spot by making an extra call.
```

Comment:
As you've mentioned below, it's an extra API call _per_ spot, so maybe worth rewording this to say `This returns the complete information for the spot by making an extra API call per returned spot.`